xml/en/docs/governance.xml

```diff
@@ -0,0 +1,83 @@
+<!--
+  Copyright (C) F5, Inc.
+  -->
+
+<!DOCTYPE article SYSTEM "../../../dtd/article.dtd">
+
+<article name="NGINX Open Source Governance"
+link="/en/docs/governance.html"
+lang="en"
+rev="1">
+<section>
+
+<para>
+Open source changed the world.
+</para>
+
+<para>
+NGINX is proud to be part of that change, from our very beginning.
+</para>
+
+<para>
+NGINX is committed to continued involvement in open source. To achieve this
+goal, we are making the following commitments that will help the longevity,
+transparency, and impact of our open source projects:
+</para>
+
+</section>
+
+<section name="Our Core Commitments">
+
+<list type="bullet">
+
+<listitem>
+We will be open, consistent, transparent, and fair in our acceptance of
+contributions.
+</listitem>
+
+<listitem>
+We will continue to enhance existing active projects
+</listitem>
+
+<listitem>
+We will release new projects that move technology forward.
+</listitem>
+
+<listitem>
+We will continue to offer projects under OSI-approved software licenses.
+</listitem>
+
+<listitem>
+We will not remove and commercialize existing projects or features.
+</listitem>
+
+<listitem>
+We will not impose limits on the use of our projects.
+</listitem>
+
+</list>
+
+</section>
+
+<section name="NGINX Open Source Principles">
+
+<list type="bullet">
+
+<listitem>
+All contributions are reviewed by the technical committee for that project
```

Comment:
punctuation in the end of the sentence should be consistent with the previous list